docs/install_guides/installing-openvino-from-archive-windows.md

```diff
@@ -178,6 +178,8 @@ To uninstall OpenVINO, follow the steps on the [Uninstalling page](uninstalling-
 
 ## Additional Resources
```

Comment:
Seems this line is duplicated with line 177.

Reply:
```suggestion
```